src/main/java/ch/njol/skript/Skript.java

```diff
@@ -1791,22 +1790,16 @@ static void logEx(final String... lines) {
 		for (final String line : lines)
 			SkriptLogger.LOGGER.severe(EXCEPTION_PREFIX + line);
 	}
-	
-	public static String SKRIPT_PREFIX = ChatColor.GRAY + "[" + ChatColor.GOLD + "Skript" + ChatColor.GRAY + "]" + ChatColor.RESET + " ";
-	
-//	static {
-//		Language.addListener(new LanguageChangeListener() {
-//			@Override
-//			public void onLanguageChange() {
-//				final String s = Language.get_("skript.prefix");
-//				if (s != null)
-//					SKRIPT_PREFIX = Utils.replaceEnglishChatStyles(s) + ChatColor.RESET + " ";
-//			}
-//		});
-//	}
-	
+
+	public static final String SKRIPT_PREFIX = "[" + ChatColor.GOLD + "Skript" + ChatColor.GRAY + "]" + ChatColor.RESET + " ";
+	private static final Message SKRIPT_PREFIX_MESSAGE = new Message("skript.prefix");
```

Comment:
We don't need two constants here when one is a fallback, I also expressed how I don't agree with fallback localization below.

Reply:
The fallback constant is fine (could probably be `private` though), for fallback localization see https://github.com/SkriptLang/Skript/pull/5437#discussion_r1105975433

Reply:
> (could probably be `private` though)

https://github.com/SkriptLang/Skript/pull/5437#discussion_r1105002295

Reply:
Umbaska seems to be the only addon using it, but sure I guess.

Reply:
I'm not against removing this value if we also go the route of changing the fallback to an empty string

Reply:
After some thoughts with myself. If what kiip1 said about only seeing Umbaska then I don't see any reason for me to keep this value and have since went ahead and just removed it